packages/libpsl/project.bri

```diff
@@ -0,0 +1,77 @@
+import * as std from "std";
+import nushell from "nushell";
+import python from "python";
+
+export const project = {
+  name: "libpsl",
+  version: "0.21.5",
+};
+
+const source = Brioche.download(
+  `https://github.com/rockdaboot/libpsl/releases/download/${project.version}/libpsl-${project.version}.tar.gz`,
```

Comment:
See https://github.com/brioche-dev/brioche-packages/pull/214#discussion_r1933182466 for the reason of using Brioche.download() here.